Directory.Packages.props

```diff
@@ -59,7 +59,7 @@
 
   <!-- Testing dependencies -->
   <ItemGroup>
-    <PackageVersion Include="Anthropic.SDK" Version="5.8.0" />
+    <PackageVersion Include="Anthropic" Version="11.0.0" />
```

Comment:
I somehow expected this to be a new package with a lower version number, but I take it's only been recently sanctioned by us?

Reply:
They're completely different packages from different authors. The previous one wasn't actually from Anthropic, despite the naming. The new one is. And for whatever reason, they chose to start at 10.